Source/Diagnostics/ComputeDiagFunctors/BackTransformFunctor.cpp

```diff
@@ -202,6 +228,64 @@ BackTransformFunctor::LorentzTransformZ (amrex::MultiFab& data, amrex::Real gamm
         amrex::Array4< amrex::Real > arr = data[mfi].array();
         amrex::Real clight = PhysConst::c;
         amrex::Real inv_clight = 1.0_rt/clight;
+#ifdef WARPX_DIM_RZ
+        const int n_rz = WarpX::n_rz_azimuthal_modes;
+        const int n_rcomps = 2*n_rz - 1;
```

Comment:
Similarly to the comment above:
```suggestion
        const int n_rcomps = WarpX::ncomps;
```